public/scripts/auctionView.js

```diff
@@ -0,0 +1,15 @@
+(function(module){
+  var auctionView = {};
+  auctionView.buttonClick = function(){
```

Comment:
I'm not totally sure you need to modularize this into it's own view and method just to assign the event handler. That being said, I also like that you did it, because it creates a clear line of separation in functionality for each of your views. 

It might be a bit overkill for what you're doing, but the applied concept is still solid!